### DISCUSSION TITLE
Running new Twill install I get Route [admin.users.index] not defined

### DISCUSSION THREAD

**Discussion by adam-jones-net** [area17/twill#1543] [Q&A] [answered]
I've installed Twill. Migrations worked but I can't login.   I get the following error 

> Route [admin.users.index] not defined

It's caused it seems due to the route function wanting _users_ table routes.   

Line 33 of vendor\area17\twill\views\partials\navigation\_overlay_navigation.blade.php

> <a href="<?php echo e(**route('admin.users.index')**); ?>"><?php echo e(twillTrans('twill::lang.nav.cms-users')); ?></a><br />

What have I missed?   Are there routes I need to setup in admin.php to make this work ?

**Comment by haringsrob** [comment]
Hey @arkid,

I do new installs all the time without issues.

You have only installed twill or also other packages?

**Comment by adam-jones-net** [comment, marked as answer]
This was installed upon an old laravel install. I'm sure something very simple was wrong but as there is so little on google about this I just gave up and have done a fresh install of everything.
